test/extensions/common/aws/credentials_provider_impl_test.cc

```diff
@@ -646,7 +646,7 @@ TEST_F(TaskRoleCredentialsProviderTest, TimestampCredentialExpiration) {
   "AccessKeyId": "new_akid",
   "SecretAccessKey": "new_secret",
   "Token": "new_token",
```

Comment:
Would you be able to include this additional unit test?
[pr_29267_test.txt](https://github.com/envoyproxy/envoy/files/12446644/pr_29267_test.txt)

Reply:
Nevermind, I think we should be good with `TimestampCredentialExpiration` test.